src/static/js/modules/jquery/cf_inputSplit.js

```diff
@@ -4,7 +4,8 @@
 
 'use strict';
 
-require( 'jquery' );
+var $ = require( 'jquery' );
+var jQuery = $;
```

Comment:
You were right to define `$` on line 7, but the self-executing function is redefining `$` [on L:11](https://github.com/cfpb/cfgov-refresh/blob/DF-1882-update-eslint/src/static/js/modules/jquery/cf_inputSplit.js#L11) using the `jQuery` object at the end. Remove that object and the definition on L:11 and you won't need to define `jQuery` on line 8. (Ping me if this still doesn't make sense).